bucket/backrest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.4.0",
+    "description": "Web UI and orchestrator for restic backup.",
+    "homepage": "https://github.com/garethgeorge/backrest",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/garethgeorge/backrest/releases/download/v1.4.0/backrest_Windows_x86_64.zip",
+            "hash": "02ca768d20315013958e889137c6e3cde157128e62745f7fa646ee2f5ebfb336"
+        },
+        "arm64": {
+            "url": "https://github.com/garethgeorge/backrest/releases/download/v1.4.0/backrest_Windows_arm64.zip",
+            "hash": "d07642972014bca45dc964c188ee83a5b4857b3cc18cfb42e782664a609e8c98"
+        }
+    },
+    "bin": [
+        "backrest.exe",
+        "backrest-windows-tray.exe"
+    ],
+    "shortcuts": [
+        [
+            "backrest.exe",
+            "Backrest"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/garethgeorge/backrest/releases/download/v$version/backrest_Windows_x86_64.zip"
+            },
+            "arm64": {
+                "url": "https://github.com/garethgeorge/backrest/releases/download/v$version/backrest_Windows_arm64.zip"
+            }
+        },
+        "hash": {
+            "url": "",
+            "regex": "$sha256\\s+$basename"
```

Comment:
```suggestion
            "url": "$baseurl/backrest_$version_checksums.txt"
```

Reply:
Should I make the necessary changes?
This is my first PR and I'm a bit confused.